README.md

```diff
@@ -96,4 +96,78 @@ query GetUsers {
 ```
 docker stop bqCore
 docker rm bqCore
-```
\ No newline at end of file
+```
+
+## **CI/CD**
+The CI/CD pipeline for platform_api is divided into three main parts:
+1. Pre-commit checks
+2. Continuous Integration (CI)
+3. Continuous Deployment (CD)
+
+Through these steps, every code update is automatically deployed to the development environment and made ready for testing.
+
+### **1. pre-commit checks:** 
+Pre-commit checks are triggered every time a local development task is completed and the code is committed via the command: `git commit`. This job includes code formatting and linting. In this repository, Prettier is used for code formatting, while ESLint is used for linting. Husky and lint-staged are utilized to apply code formatting and linting to files that are staged in Git (via the git add command).
```

Comment:
For `pre-commit checks`, technically it's not part of the CI/CD pipeline, as it take place before code is pushed to the repository